validators/watchdog.adoc

```diff
@@ -0,0 +1,121 @@
+= Watchdog
+:doctype: article
+:icons: font
+:toc: macro
+
+ifdef::env-github[]
+
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+
+toc::[]
+
+endif::[]
+
+== Introduction
+
+SKALE watchdog is microservice running on every SKALE node for providing SLA agents with node performance metrics. It also can be used for external monitoring goals (validators can use it in their own monitoring tools).
+
+Watchdog is implemented as a Python script running in docker container with Flask webserver that provides simple REST JSON API available on port 3009 (http://YOUR_SKALE_NODE_IP:3009). Currently watchdog can provide data about all SKALE node docker containers, healthchecks of sChains and SGX server, ethereum node endpoint status and hardware information.
+
+== REST JSON API
+
+=== /status/core
+
+==== Description
+
+Returns data about all SKALE containers running on a given node. These are containers with prefixskale_, including base containers and pairs of sChain and IMA containers, where every pair is corresponding to a certain sChain assigned to this node. 
+
+Here’s a list of the base containers:
+
+* skale_transaction-manager
+* skale_admin
+* skale_api
+* skale_mysql
+* skale_sla
+* skale_bounty
+* skale_watchdog
+* skale_nginx
+
+Docker container name patterns for sChain with name SCHAIN_NAME are the following:
+
+* sChain: skale_schain_SCHAIN_NAME
+* IMA: skale_ima_SCHAIN_NAME
+
+==== Usage
+
+```shell
+$ curl http://YOUR_SKALE_NODE_IP:3009/status/core
```

Comment:
Can we add an explanation for what it returns instead of full data? maybe we can explain what it returns like List of all containers, container versions, the status of each container.

Reply:
my bad I didn't see the previous explanation part